src/plugins/navigation/public/top_nav_menu/top_nav_menu_data.tsx

```diff
@@ -29,3 +29,7 @@ export interface TopNavMenuData {
   disableButton?: boolean | (() => boolean);
   tooltip?: string | (() => string);
 }
+
+export interface RegisteredTopNavMenuData extends TopNavMenuData {
```

Comment:
I suspect you have some unrelated commits in this PR

Reply:
Indeed. I wrote in #kibana about this boo boo, trying to fix it now.

Sorry for the inconvenience